docs/how-to-guides/deployment-options.md

```diff
@@ -0,0 +1,160 @@
+
+(**TODO** mention default deployment settings, i.e., auth disabled but recommend enabling, template store created, app insights created, etc.)
+
+#### Configurable settings
+
+To learn more about the various options available to customize your service, and to configure the settings of your FHIR converter service in Azure, refer [Configure FHIR converter service settings](configuration-settings.md).
+
+### Deployment options
+
+#### Option 1: Single-click Deploy to Azure via ARM template generated from Bicep Template
+
+#### Option 2: Deploy a single Bicep file locally
+
+Deploy the [Single Deploy Bicep Template](../deploy/FhirConverter-SingleAzureDeploy.bicep) by running the following command:
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep
+```
+
+Note: See [region availability](https://azure.microsoft.com/en-us/explore/global-infrastructure/products-by-region/?products=monitor,storage,container-apps) for the required resources to select a valid location for the resources to be deployed in.
+
+You will need to provide a *serviceName* that will be used to generate a name for each of the resources provisioned, and the *containerAppImageTag*, which is the tag of the FHIR Converter image version to be pulled from MCR. To see available image tags, visit the [FHIR Converter MCR page](https://mcr.microsoft.com/en-us/product/healthcareapis/fhir-converter/tags).
+
+You have the option specify custom values for any of the resources created by adding parameters to the command. For example, the containerAppName can be customized to be 'containerapp-test1' by specifying a value for the containerAppName parameter in the command:
+```
+az deployment sub create --location westus3 --template-file FhirConverter-SingleAzureDeploy.bicep --parameters containerAppName=containerapp-test1
+```
+
+For help, type '?' to see a description of a parameter.
+
+By default, the Single Deploy Bicep Template will result in a FHIR Converter deployment with the following settings:
+
+
+The deployment will create an Application Insights instance that will receive application logs and metrics for the FHIR Converter service. See the [Monitoring Overview](monitoring.md) for more information on how to view these logs and metrics.						
+To disable Application Insights deployment for your service, or if you initially deployed your service with Application Insights and now want to disable telemetry export to Application Insights, run the deployment command with `--parameters deployApplicationInsights=false` included:
+
+```
+az deployment sub create --location <Location> --template-file FhirConverter-SingleAzureDeploy.bicep --parameters deployApplicationInsights=false
+```
+
+Note: for any time when the service is running while Application Insights is disabled, you will not have access to metrics and request logs that were captured during that time.
+
+**2. Security settings for the API endpoints are disabled.**
```

Comment:
Should mention either here or in a separate deployment section in the troubleshooting doc that the array for security audiences needs to be of the format '["...","..."]'